src/OpenTelemetry.AutoInstrumentation.AdditionalDeps/Directory.Packages.props

```diff
@@ -0,0 +1,12 @@
+<Project>
```

Comment:
Any reason it's separate from the main `Directory.Packages.props` in src dir?

Reply:
I just mirrored the behavior of `src\OpenTelemetry.AutoInstrumentation.AdditionalDeps\Directory.Build.props`. The behavior is:

The existence of file `src\OpenTelemetry.AutoInstrumentation.AdditionalDeps\Directory.Build.props` would end MSBuild's search for `Directory.Build.props` and the file not import `src\Directory.Build.props`.
Thus  `src\Directory.Build.props` won't be applied to AdditionalDeps project.